### PR TITLE
#149954569 Comment on contribution

### PR DESCRIPTION
**Pull request by KaiserPhemi**
### What does this PR do?
- Create the contribution comment user interface.

#### Description of Task to be completed
- User should be able to add comments to a contribution

#### How should this be manually tested?
- Pull the branch to local machine
- `yarn install` to install latest dependencies
-  `yarn run start:dev` to run the application
- Navigate to `http://calm-dev.andela.com:3000/contributions/:id/comments`

#### Any background context you want to provide?
- NA

#### Relevant Pivotal Tracker stories
- [149954569](https://www.pivotaltracker.com/n/projects/2059149/stories/149954569)

#### Screenshots (if appropriate)
screen shot 2017-08-24 at 7 01 08 am
screen shot 2017-08-24 at 7 01 31 am

### CODE REVIEW

File: src/components/comment/Comment.scss [hunk 1]
```diff
@@ -32,10 +30,8 @@
   .comment-form {
     align-items: center;
     display: flex;
```
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: display then align, let's rearrange it so it looks like one person coding

File: src/components/comment/Comment.scss [hunk 2]
```diff
@@ -48,7 +44,7 @@
 
       .comment-content{
         border: 0;
-        border-bottom: 1px solid #dedede;
+        color: #222222;
         font-size: 1rem;
         font-weight: 100;
         height: 1.8rem;
```
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: Height should come first, lets change that

File: src/components/comment/Comment.scss [hunk 3]
```diff
@@ -60,32 +56,43 @@
...
     height: auto;
     margin: 10px 0;
+    list-style: none;
     padding: 0.625rem;
     
+    .comment-reply {
+      margin: 3px;
+      margin-left: 20%;
+      height: 2.8rem;
+      width: 80%;
```
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: width, height and margin

File: src/components/comment/Comment.scss [hunk 4]
```diff
@@ -107,9 +114,9 @@
   }
 
   .load-more-btn {
+    cursor: pointer;
```
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: Cursor should come last here

File: src/components/comment/Comment.scss [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
 
 .comment-container {
-  width: 28.29375rem;
+  width: 36.38rem;
   margin: 0 auto;
-  margin-top: 230px;
-  $content-width: 36.38rem;
 
   .user-dp {
     background-color: #000;
```
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: Can we use the theme.scss for this ? what do you think ?

File: src/components/comment/Comment.scss (hunk 1 above)
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: Let display come first

File: src/components/comment/Comment.scss [hunk 6]
```diff
@@ -60,32 +56,43 @@
   .comment-list {
     border: 1px solid #dedede;
     border-radius: 4px;
-    width: $content-width;
     height: auto;
```
Comment (andela-cofor) [inline src/components/comment/Comment.scss]: height should also come first...

File: src/components/comment/CommentForm.component.tsx [hunk 7]
```diff
@@ -1,16 +1,36 @@
...
   }
+  private onChange(event) {
+    return this.setState({userComment: event.target.value});
+  }
+  private onSubmit(event) {
+    event.preventDefault();
+  }
 }
 
 export default CommentForm;
```
Comment (andela-cofor) [inline src/components/comment/CommentForm.component.tsx]: Why are we using two export statements ?

File: src/components/comment/CommentCard.component.tsx [hunk 8]
```diff
@@ -16,16 +56,14 @@ class CommentCard extends React.Component<any, any> {
...
+            comments={this.state.allComments}
+          />
         </div>
       </div>
     );
   }
-
 }
 
 export default CommentCard;
```
Comment (andela-cofor) [inline src/components/comment/CommentCard.component.tsx]: Why are we using two export statements ?

File: src/components/comment/CommentCard.component.tsx [hunk 9]
```diff
@@ -6,8 +6,48 @@ import './Comment.scss';
 import CommentForm from './CommentForm.component';
 import CommentList from './CommentList.component';
```
Comment (andela-cofor) [inline src/components/comment/CommentCard.component.tsx]: I notice you are not using this `import { Button } from 'react-toolbox/lib/button';` you can take it off

File: src/components/comment/Comment.scss (hunk 1 above)
Comment (KaiserPhemi) [inline src/components/comment/Comment.scss]: Fixed

File: src/components/comment/CommentForm.component.tsx (hunk 7 above)
Comment (KaiserPhemi) [inline src/components/comment/CommentForm.component.tsx]: Fixed

File: src/components/comment/CommentCard.component.tsx (hunk 8 above)
Comment (KaiserPhemi) [inline src/components/comment/CommentCard.component.tsx]: Fixed

File: src/components/comment/CommentCard.component.tsx (hunk 9 above)
Comment (KaiserPhemi) [inline src/components/comment/CommentCard.component.tsx]: Done.

File: src/components/comment/Comment.scss (hunk 6 above)
Comment (KaiserPhemi) [inline src/components/comment/Comment.scss]: Noted and fixed.

File: src/components/comment/Comment.scss (hunk 3 above)
Comment (KaiserPhemi) [inline src/components/comment/Comment.scss]: Fixed

File: src/components/comment/Comment.scss (hunk 1 above)
Comment (KaiserPhemi) [inline src/components/comment/Comment.scss]: Noted and fixed.

File: src/components/comment/Comment.scss (hunk 5 above)
Comment (KaiserPhemi) [inline src/components/comment/Comment.scss]: Sure. This has been refactored to use variables from `theme.scss`.

File: src/common/LoadMoreButton/LoadMoreButton.component.scss [hunk 10]
```diff
@@ -0,0 +1,16 @@
+@import '../../common/theme.scss';
+
+.load-more-btn {
+  width: 100%;
+  border: 1px solid $border-color;
+  border-radius: 4px;
+  background-color: white;
+  font-weight: 300;
+  text-transform: capitalize !important;
+  color: map-get($grey, 550) !important;
```
Comment (andela-cofor) [inline src/common/LoadMoreButton/LoadMoreButton.component.scss]: I think we should put color before font-weight, what do you think ?

**Pull request merged**